expat/tests/Makefile.am

```diff
@@ -64,3 +64,7 @@ EXTRA_DIST = \
     udiffer.py \
     xmltest.log.expected \
     xmltest.sh
+
+if BUILD_FUZZTARGETS
+    SUBDIRS += fuzz
+endif
```

Comment:
It would be nice to have that code close to `SUBDIRS = . benchmark` at the start of the file.